nixos/tests/initial-home-contents.nix

```diff
@@ -0,0 +1,27 @@
+import ./make-test.nix ({ pkgs, ...} : {
+  name = "initial-home-contents";
+  meta = with pkgs.stdenv.lib.maintainers; {
+    maintainers = [ bobvanderlinden ];
+  };
+
+  nodes = {
+    machine = { config, lib, pkgs, ... }: {
+      users.users.testuser = {
+        isNormalUser = true;
+        home = "/home/testuser";
```

Comment:
Should this have `createHome` set to `true` here?

Asking because of: https://github.com/bobvanderlinden/nixpkgs/blob/7488d5be2256473f65e78e978362bac31bbb7858/nixos/modules/config/update-users-groups.pl#L215

I could be wrong.

Reply:
`isNormalUser` implies `createHome` as well: https://github.com/bobvanderlinden/nixpkgs/blob/e406c2d95d497da3ca6d05ac0c0a84188d372bed/nixos/modules/config/users-groups.nix#L280-L286

Reply:
Ahh, my bad.